src/components/CurriculumList.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <section class="screen-box">
+    <Header
+      title="カリキュラム一覧"
+      url="/"
+    />
+    <main class="screen-box__main curriculum-list">
+      <div
+        v-for="(curriculum, index) in curriculumList"
+        :key="index"
+        :style="styleObject( curriculum )"
+        class="curriculum-box"
+      >
+        <p>{{ curriculum.title }}</p>
+        <div
+          class="curriculum-box__subcontent"
+          @click="$router.push(`/curriculum/${index}`)"
+        >
+          <label class="received">{{ curriculum.receiveDate }}</label>
+          <label class="coach-name">{{ curriculum.coachName }}</label>
+        </div>
+      </div>
+    </main>
+  </section>
+</template>
+
+<script>
+import Header from './Header.vue'
+
+export default {
+  name: 'CurriculumList',
+  components: { Header },
+  data () {
+    return {
+      curriculumList: {},
+    }
+  },
+  created () {
+    this.curriculumList = this.mock()
+  },
+  methods: {
+    onClickClose () {
+      this.$router.push('/')
+    },
+    styleObject (curriculum) {
+      return {
+        background: `url("${curriculum.thubnailUrl}") no-repeat`,
+        'background-position': 'center center',
+        'background-size': 'contain',
+        width: `${window.innerWidth-40}px`,
```

Comment:
`window.innerWidth-40` should be defined as variable

Reply:
As you say. I do not understand the meaning of 40 with this.
Modify it to correct what you defined.